common/views/components/SearchForm/SearchForm.SortByPortal.tsx

```diff
@@ -1,26 +0,0 @@
-import { FunctionComponent, ReactElement, useEffect, useState } from 'react';
-import { createPortal } from 'react-dom';
```

Comment:
It looks like this is the last use of `react-dom` in the project, so let's remove it from `package.json` also.

Reply:
I believe it's part of Next's build setup and is needed to run the project in general (paired with `react`). It won't build without. But good shout in checking dependencies, I'll have a look through!